x-pack/solutions/security/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/upgrade_prebuilt_rules_table/use_upgrade_with_conflicts_modal/translations.tsx

```diff
@@ -41,202 +43,139 @@ export const UPGRADE_RULES_WITH_CONFLICTS = (numOfRules: number) =>
-      values={{
-        numOfRules: numOfRulesWithoutConflicts + numOfRulesWithSolvableConflicts,
-        numOfRulesStrong: (
-          <strong>{numOfRulesWithoutConflicts + numOfRulesWithSolvableConflicts}</strong>
-        ),
-        numOfRulesWithSolvableConflictsStrong: <strong>{numOfRulesWithSolvableConflicts}</strong>,
-      }}
-    />
-    <br />
-    <br />
-    {PROCEED_WITH_CONFLICT_FREE_RULES(numOfRulesWithoutConflicts)}
-    <br />
-    {PROCEED_WITH_CONFLICT_FREE_AND_SOLVABLE_CONFLICT_RULES({
-      numOfRulesWithoutConflicts,
-      numOfRulesWithSolvableConflicts,
-    })}
-  </>
-);
+  return (
+    <>
+      <div>
+        <FormattedMessage
+          id="xpack.securitySolution.detectionEngine.upgradeConflictsModal.rulesWithAutoResolvedConflictsGuidance"
+          defaultMessage="{numOfRulesWithSolvableConflictsStrong} {numOfRulesWithSolvableConflicts, plural, =1 {rule} other {rules}} with auto-resolved {numOfRulesWithSolvableConflicts, plural, =1 {conflict} other {conflicts}} can still be updated. Choose one of the following options:"
+          values={{
+            numOfRulesWithSolvableConflicts,
+            numOfRulesWithSolvableConflictsStrong: (
+              <strong>{numOfRulesWithSolvableConflicts}</strong>
+            ),
+          }}
+        />
+        <ul>
+          <li>
+            <FormattedMessage
+              id="xpack.securitySolution.detectionEngine.upgradeConflictsModal.useRuleUpdateFlyout"
+              defaultMessage="Use the rule update flyout to address auto-resolved conflicts. This is the safest option and gives you more control over the final update. {learnMoreLink}"
+              values={{
+                learnMoreLink: (
+                  <EuiLink href={docsUrl} target="_blank">
+                    <FormattedMessage
+                      id="xpack.securitySolution.detectionEngine.upgradeConflictsModal.learnMore"
+                      defaultMessage="Learn more"
+                    />
+                  </EuiLink>
+                ),
+              }}
+            />
+          </li>
+          <li>
+            <FormattedMessage
+              id="xpack.securitySolution.detectionEngine.upgradeConflictsModal.bulkUpdateRules"
+              defaultMessage="Click {updateRules} to bulk-update rules with auto-resolved conflicts and rules without conflicts."
+              values={{
+                updateRules: (
+                  <strong>
+                    {UPGRADE_RULES_WITH_CONFLICTS(
+                      numOfRulesWithSolvableConflicts + numOfRulesWithoutConflicts
+                    )}
+                  </strong>
+                ),
+              }}
+            />
+          </li>
+          <EuiCallOut
+            title={ACCEPT_SOLVABLE_CONFLICTS_WARNING}
+            color="warning"
+            iconType="warning"
+          />
+        </ul>
+      </div>
+      <br />
+    </>
```

Comment:
nit: this seems like more jsx than we'd usually put in a `translations` file (eui elements, hooks, etc.), might be worth making its own component and just keeping the messages in here.